handler/src/main/java/io/netty5/handler/ssl/ConscryptAlpnSslEngine.java

```diff
@@ -41,21 +44,22 @@
 /**
  * A {@link JdkSslEngine} that uses the Conscrypt provider or SSL with ALPN.
  */
-abstract class ConscryptAlpnSslEngine extends JdkSslEngine {
+abstract class ConscryptAlpnSslEngine extends JdkSslEngine implements VectoredUnwrap {
     private static final boolean USE_BUFFER_ALLOCATOR = SystemPropertyUtil.getBoolean(
             "io.netty5.handler.ssl.conscrypt.useBufferAllocator", true);
 
-    static ConscryptAlpnSslEngine newClientEngine(SSLEngine engine, ByteBufAllocator alloc,
+    static ConscryptAlpnSslEngine newClientEngine(SSLEngine engine, io.netty5.buffer.api.BufferAllocator alloc,
             JdkApplicationProtocolNegotiator applicationNegotiator) {
         return new ClientEngine(engine, alloc, applicationNegotiator);
     }
 
-    static ConscryptAlpnSslEngine newServerEngine(SSLEngine engine, ByteBufAllocator alloc,
+    static ConscryptAlpnSslEngine newServerEngine(SSLEngine engine, io.netty5.buffer.api.BufferAllocator alloc,
```

Comment:
nit: import `BufferAllocator` ?

Reply:
Conscrypt has their own thing called `BufferAllocator`, so one of them needs to be qualified.